tutorials/02a_RunningREMINDLocally.md

```diff
@@ -1,10 +1,146 @@
 # Running REMIND locally
-Mika Pflüger (mika.pflueger@pik-potsdam.de)
+Mika Pflüger (mika.pflueger@pik-potsdam.de), Tonn Rüter (tonn.rueter@pik-potsdam.de)
 
-*Note:* This tutorial is a work in progress, not all steps needed are described, yet.
+*Note:* This approach of running REMIND is a work in progress, however successful REMIND runs on local machines have been conducted using this tutorial.
+
+## System Requirements
+
+- Installed python version must be `3.10.X` or older. To check your python version run `python --version` in a terminal of your choice
+- GAMS must be installed and the path to GAMS applications needs to be in your PATH environment variable
+- R version must be at least `4.2.X`. To check your R version run `R --version` in a terminal of your choice
+- On Windows we recommend to use PowerShell rather than the standard command prompt
+
+## Use Local Input Data
+
+**This is the supported way of input data handling**
+
+As long as we do not provide a curated collection of input data please obtain these files from the cluster:
+
+```bash
+rev6.606_62eff8f7_remind.tgz
+rev6.606_62eff8f7_validationremind.tgz
+CESparametersAndGDX_878ac5d69254efb4eba5c1fa39aba64000307bb1.tgz
+```
+
+The contents of these tar-archives are necessary to run REMIND locally. Since REMIND input data is constantly updated the up-to-date file names may vary, but will exhibit a similar pattern. The first two files can be found on the cluster at `/p/projects/rd3mod/inputdata/output`, the latter file in `/p/projects/remind/inputdata/CESparametersAndGDX`.
+
+Download & store them in a folder of your choice. The corresponding path is provided to REMIND via environment variable. Add to your local `~/.Renviron` file (in linux, found in your home directory, in windows, found at `C:\Users\<your windows username>\Documents\.Renviron`):
+
+```bash
+# REMIND data setup
+# If the input data is spread over multiple directories,
+# separate them with a : on linux, a ; on windows
+REMIND_repos_dirs="/my/first/path:/my/second/path"
+# On windows, it would look something like
+# REMIND_repos_dirs="C:\Users\<your windows username>\REMIND;D:\REMIND"
+# depending on your used file paths
+```
+
+You do not have to unpack tar-archives as this will be done during the start-up of your REMIND run.
+
+## Start a run
+
+In a terminal session change directory to the location where REMIND was cloned via `cd <REMIND_DIR>` and start a run as you would do on the cluster. As a linux user, type
+
+```bash
+Rscript start.R
+```
+
+or as a Windows user, type
+
+```PowerShell
+Rscript.exe .\start.R
+```
+
+## Known Issues & Further Debugging
+
+### MAGCFG_STORE File Path Resolution Fails in `prepare.R`
```

Comment:
This point should be solved by setting
`cfg$magicc_template` in `default.cfg` to  <- "[PATH TO REMIND]/core/magicc/"

Reply:
Works 👍 

I removed this from the debugging section and added another in which I explain how to adapt the `default.cfg` file